content/References/dRAIDprimer.md

```diff
@@ -45,24 +54,24 @@ When datasets are expected to have a heavy sequential I/O pattern, a 1MB record
 However, datasets, and typically zvols, are expected to have heavy random I/O patterns, it is recommended to remain closer to the minimum 128k (or 64k for zvols) record size.
 Selecting a record/block size smaller than the minimum allocation size is catastrophic for pool capacity.
 
-#### Permutation maps
+#### Permutation Maps
 
 dRAID uses an array of predetermined permutation maps to determine where data, parity, and spare capacity reside across the pool.
 This ensures that during resilvers, all IO (reads and writes) distribute evenly across all disks, reducing the load on any one disk.
 
 Because a permutation map automatically selects during pool creation, distributed spares cannot be added after pool creation.
-If adding spares after pool creation is a critical requirement, create the pool using a raidz layout.
+If adding spares after pool creation is a critical requirement, create the pool using a RAIDz layout.
 
-#### Distributed hot spare
+#### Distributed Hot Spare
 
-dRAID uses a different methodology from raidz to provide hot spare capacity to the data vdev.
+dRAID uses a different methodology from RAIDz to provide hot spare capacity to the data vdev.
 Distributed hot spares are allocated as blocks on each disk in the pool.
 This means that hot spares must be allocated during vdev creation, cannot be modified later, and that every disk added to the vdev is active within the storage pool.
 
 Disk failure results in the dRAID vdev copying the parity and data blocks onto the spare blocks in the each fixed stripe width disk.
```

Comment:
"spare blocks in the each fixed stripe width disk." This part reads poorly, "in the each" needs to be changed.

Reply:
removed "the" which should not be here in this sentence so it now reads "in each fixed stripe width disk"